source/guide_mattermost.rst

```diff
@@ -160,12 +171,94 @@ When upgrading to Mattermost 6.4 or newer you need to change the collation of th
   [isabell@stardust ~]$ mysql -e "ALTER DATABASE isabell_mattermost COLLATE = utf8mb4_general_ci;"
   [isabell@stardust ~]$
 
+Automatic Update
+-----------------
+
+Use the script attached to update Mattermost to the current version. Run the script above the mattermost-root directory.
+
+.. code-block:: shell
+
+	#!/bin/sh
+	printf "\n===\nWelcome to updating Mattermost... let us begin!\n===\n"
+
+	printf "\n== Preflight Checks ==\n"
+	if [ ! -d "./mattermost" ]; then
+	printf "Error: ./mattermost directory does not exist. Are you running in the correct place?. Exit\n"
+	exit 1
+	else 
+		printf "Directory ./mattermost found.\n"
+	fi
+
+	database=$(cat ./mattermost/config/config.json | jq '.SqlSettings.DataSource' | grep -o '/.*?' | tail -c +2 | head -c -2)
+	if [ -z "$database" ] 
+	then
+		printf "\nError: Could not extract the database name from ./mattermost/config/config.json. Maybe the script runs in the wrong directory? Exit 1.\n"
+		exit 2
```

Comment:
It says `Exit 1` but it does `exit 2`.
You can just say something like `Could not ... - Abort update procedure` and if you want `with exit code 2`

And just to make it more simple you can extract all the formatting stuff to a helper function.
This example also redirects the error messages so users are able to filter the important error messages and ignore the info messages.
```bash
function error_message { echo "Error: $*" >>/dev/stderr }

# And then just use it like this
error_message "Houston, we have a problem - Abort update procedure with exit code 42"
```